src/components/templates/Auditorium/Auditorium.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+
+import { SectionPreviews } from "./components/SectionPreviews";
+import { Section } from "./components/Section";
+
+import { AuditoriumVenue } from "types/venues";
+
+const enum AuditoriumVideoPosition {
+  TOP = "top",
+  CENTER = "center",
+}
+
+export interface AuditoriumProps {
+  venue: AuditoriumVenue;
+}
+
+/*
+  TODO: Sections will be stored as a collection of a Auditorium venue.
+  Each should have its own autogenerated id.
+  Each will contain chats.
+
+  I guess we should have more stricter view choice. For example: 6, 10, 20, 30+ sections.
+  This way we can provide best UI for each(most used) case
+*/
+
+export const Auditorium: React.FC<AuditoriumProps> = ({ venue }) => {
+  // self-explainatory name. I wonder if I should have this both for FULL and SECTION view?
```

Comment:
Should it be possible to position video differently for section previews page and section page?

Reply:
Hrmm, fair enough.. I don't think i've ever thought too deeply about that aspect.. my initial gut feel says "probably not", but I haven't really enumerated cases where that might be a useful thing to have.

Part of me wants to say that the position is possibly only really relevant at the 'section' view.. but then that could feel 'disconnected'; an example:

- On the 'main zoomed out view' we have the 'main embedded content' in the centre, with sections all around the edges (similar to our current `Audience` layout).
- I click on a section on the left hand side of the 'main content'
- If we render things 'the same at both levels', then the seating within my section will wrap around the entire 'main embedded content'; yet I would be somewhat expecting to be 'sitting on the left hand side' of it.

I guess we need to think about how closely we want to match how it would work 'in the real world', vs 'taking conceptual liberties' given we're in a digital platform.

@mobilevinay @sparkleed @sparkletown/product I feel like this is a question we probably need to ask/answer at the 'product level'.

Reply:
(raised in https://github.com/sparkletown/internal-sparkle-issues/issues/500#issuecomment-796386164 as well)

Reply:
> Should it be possible to position video differently for section previews page and section page?

No. This is the kind of stuff we have no real evidence actually improves the experience. But not doing it keeps things much simpler and I'm inclined to keep it simple. Once we've actually done 100+ events with this auditorium and we are HEARING this sort of feedback, we can look into it.

I think the auditorium experience can be enhanced in so many other ways - this is not an obvious big deal. so please keep things centered... :)

Reply:
> so please keep things centered

@mobilevinay We've spoken about 'center top' (similar to Jazz Bar) and 'actually centered' (as the current `Audience` component is) for these new designs so far. Can you please confirm that what you said above (and quoted in this comment) doesn't invalidate/change that, and that we still want to support both of these modes? (though having them apply to both the 'outer' and 'section' views, as you mentioned above)

Reply:
in my comment - I meant horizontally centered. We do want to support 2 types of vertical modes to allow for "all around" and "screen in front, seating around, but not all the way" modes.